01-main/packages/freeplane

```diff
@@ -4,7 +4,9 @@ if [ "${ACTION}" != "prettylist" ]; then
     # Find the latest .deb URL for the current naming pattern
     local REDIR_URL=$(grep -m 1 -Eo  'https://sourceforge.net/projects/freeplane/files/freeplane%20stable/freeplane_[1-9].*\.deb/download' $CACHE_FILE )
     URL=$(unroll_url "${REDIR_URL}")
-    VERSION_PUBLISHED="$(echo "${URL}" | grep -oP 'freeplane_\K([0-9]+\.[0-9]+\.[0-9]+(~upstream-[0-9]+)*)(?=_all\.deb)')"
+    VERSION_PUBLISHED="$( grep -oP 'freeplane_\K([0-9]+\.[0-9]+\.[0-9]+(.upstream-[0-9]+)*)(?=_all\.deb)' <<<${URL} )"
```

Comment:
```suggestion
    VERSION_PUBLISHED=$( grep -oP 'freeplane_\K([0-9]+\.[0-9]+\.[0-9]+(.upstream-[0-9]+)*)(?=_all\.deb)' <<<"${URL}" )
```
Quotes are never needed around `$()` usage (though they don't hurt).
Quoting URL in case it has shell metacharacters in it.